perfscale_regression_ci/utils/run_workload.sh

```diff
@@ -12,9 +12,13 @@ cleanup(){
 }
 
 run_workload(){
-    setup
+    if [[ ! -d e2e-benchmarking/workloads/kube-burner ]]; then
+        setup
+    fi
     cd e2e-benchmarking/workloads/kube-burner
-    ./run.sh
+    ./run.sh | tee "kube-burner-$(date +%Y%m%d%H%M%S).out"
```

Comment:
think you want an & sign here like, this way it'll print to output and stdout at the same time https://github.com/openshift-qe/ocp-qe-perfscale-ci/blob/8054c778ef44ba21bc278e3f2ff08339b6bfa0fa/Jenkinsfile#L317

Reply:
@paigerube14 You're right. Added. Please review again.